main.tf

```diff
@@ -97,3 +97,36 @@ resource "azurerm_mariadb_configuration" "config" {
   resource_group_name = azurerm_resource_group.mariadb_rg.name
   server_name         = azurerm_mariadb_server.mariadb_server.name
 }
+
+########################################
+# Monitoring
+########################################
+resource "azurerm_monitor_metric_alert" "mariadb" {
+  for_each            = var.monitor_metric_alert_criteria
+  name                = "${local.name}-${upper(each.key)}"
+  resource_group_name = azurerm_resource_group.mariadb_rg.name
+  scopes              = [azurerm_mariadb_server.mariadb_server.id]
+  tags                = module.tags.tags
```

Comment:
Should we be doing `tags_no_name` here? Not sure how the Name tag interacts with the `name` attribute in Azureland.

Reply:
Yep, I switched to setting the local.tags and to tags_no_name and referencing that